frontend/src/hooks/useDrawerWidth.ts

```diff
@@ -0,0 +1,20 @@
+import { RefObject, useLayoutEffect, useState } from 'react'
+
+export const useDrawerWidth = (drawerRef: RefObject<HTMLDivElement>) => {
```

Comment:
should we add documentation for the purpose of this hook?

Reply:
decided not to write own hook and use useMeasure form react-use instead, it should be much clearer now